### DISCUSSION TITLE
JointJS & React

### DISCUSSION THREAD

**Discussion by CodeSippr** [clientIO/joint#2299] [Q&A]
### Introduction

I have a question about the interaction between JointJS and ReactJS. A few releases ago, JointJS introduced the createRoot method to replace ReactDOM.render(). In my project I have two dia.Paper between which a drag and drop function is implemented. Unfortunately, this drag-and-drop functionality no longer works with the new createRoot method. Has anyone else encountered problems with this new method or know what could be the cause? 
Here is my original index.ts: 
```
ReactDOM.render(
      <App />
  document.getElementById("root")
);
```
and here is the new index.ts:
```
const container = document.getElementById("root");
const root = createRoot(container!);
root.render(<App />);
```

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by jamesgeorgewilliams** [comment]
`createRoot` is a React specific API introduced for the latest version of React which is v18 at the time of writing.
https://react.dev/blog/2022/03/08/react-18-upgrade-guide

Could you create a minimal example online which reproduces the issue?
https://codesandbox.io/
https://stackblitz.com/
